typing/includeclass.ml

```diff
@@ -102,9 +102,9 @@ let include_err ppf =
   | CM_Hide_virtual (k, lab) ->
       fprintf ppf "@[The virtual %s %s cannot be hidden@]" k lab
   | CM_Public_method lab ->
-      fprintf ppf "@[The public method %s cannot become private" lab
+      fprintf ppf "The public method %s cannot become private" lab
   | CM_Virtual_method lab ->
-      fprintf ppf "@[The virtual method %s cannot become concrete" lab
+      fprintf ppf "The virtual method %s cannot become concrete" lab
```

Comment:
This is clearly a good fix, but I think that it would be even better to pack every of those messages (those that are just `fprintf ppf ...`, not the other ones) in a box (`@[...@]`), rather than removing the box completely. The balancing bug appears to have been introduced in b96208b7a247cbb6d9d162fbfaf54448af33589c, and it seems that the previous code did have boxes on all messages.

Reply:
Indeed, now I see that initially there were boxes here, so probably, the better solution would be to return them back.